README.md

```diff
@@ -142,6 +142,11 @@ some caveats which will need accommodations:
     now is to add `!important` to conflicting properties in your `:host` rule.
     See [#147](https://github.com/oddbird/popover-polyfill/issues/147) for more.
 
+- When supported, the polyfill creates a cascade layer named `popover-polyfill`.
+  If your styles are not in layers then this should have no impact. If your
+  styles do use layers, you'll need to ensure the polyfill layer is declared
+  first. (e.g. `@layer popover-polyfill, other, layers;`)
+
```

Comment:
(Non blocking question) Can layers be re-used? Would it make sense to have `@layer polyfill {}` instead, in the hopes that it can become the defacto layer to put CSS polyfills in?

Reply:
That's actually a really good call. Yes, they can be extended and they can have hierarchy. Maybe `polyfill.popover` would be best. That gives consumers the chance to set a layer order including `polyfill` alone, but we can also set an order for specific polyfills, if needed.

@shannonmoeller also had thoughts about later naming conventions for public packages.

Reply:
I thought it might make sense to mirror npm or repo namespacing in the layers to ensure uniqueness and manageability. Maybe something like one of these:

- `@layer npm.oddbird.popover-polyfill`
- `@layer github.oddbird.popover-polyfill`
- `@layer vendor.oddbird.popover-polyfill`

Reply:
Ah, I'm glad I asked, but I think that probably won't work because it would force all the `npm.*` or `oddbird.*` styles to be grouped together in the layer order. It seems reasonable to me to group all `polyfills.*` together, but even that might be presumptuous without some namespacing... 

Does anyone have thoughts about `oddbird-polyfills.popover` vs `polyfills.popover` to avoid potential naming conflicts?

Reply:
Great points. I think `oddbird-polyfills.popover` is pretty unique. Though `polyfills.popover` seems good too. If people are including more than one popover polyfill on the page which would conflict with that I think they have a bigger issue to deal with.

Reply:
Whatever we call our internal layers, consuming authors can create their own groupings and avoid name conflicts by nesting on their end. I don't think we should worry about trying to do those things for them. I'd say `popover-polyfill` on it's own is probably plenty, and we don't even need the sublayer I suggested in the issue.

Layer-naming-by-vendor/source conflates two things that aren't really related in my mind. But whatever we do, authors can pull these styles into an `oddbird` layer if they choose to.

Reply:
Oh that's true in this case. I'm not used to injecting styles on someone else's site 😅 - but yeah, I think the simpler solution still makes sense.